reference/features/known_limitations.md

```diff
@@ -1,27 +1,24 @@
 # Known limitations
 
-MeiliSearch has known limitations, some of which may be addressed in future versions.
-
-Some of these limitations have been made by MeiliSearch developers for relevancy and/or performance, while others are enforced by [LMDB](/reference/under_the_hood/storage.md), the key-value store used by MeiliSearch.
+Currently, MeiliSearch has a number of known limitations. Some of these limitations are the result of design trade-offs when balancing relevancy and performance, while others are dictated by [LMDB](/reference/under_the_hood/storage.md), the key-value store we use under the hood.
 
 ## Design limitations
 
 ### Number of query words
 
-The maximum number of words taken into account for each [search query](/reference/features/search_parameters.md#query-q) is 10. If a search query includes more than 10 words, all words after the 10th will be ignored.
+The maximum number of terms taken into account for each [search query](/reference/features/search_parameters.md#query-q) is 10. **If a search query includes more than 10 words, all words after the 10th will be ignored.**
 
-This is because large queries can lead to long response times, and MeiliSearch's goal is always to provide a [fast type-as-you-search experience](/learn/what_is_meilisearch/philosophy.md#front-facing-search).
+Large queries can lead to long response times and MeiliSearch's goal is to provide a [fast search-as-you-type experience](/learn/what_is_meilisearch/philosophy.md#front-facing-search).
```

Comment:
After your changes, I find this sentence a bit confusing:

> Large queries can lead to long response times

As we just said, there's a limit of 10 query terms max, making it impossible for users to make large queries. But this sentence makes it sound as though it may be possible (but merely discouraged).

This could be solved by adopting a "Limitation: ... Explanation: ..." format, as described in a separate comment.

Reply:
Hmmm…my interpretation of `large queries` is not that the query _contains_ many terms, but that it _returns_ many documents.

This confusion means that I need to 1) confirm what is the correct interpretation of the original sentence and 2) rephrase it anyway to avoid misunderstandings.

Reply:
Ignore my previous comment as it makes no sense.

The limitation/explanation format helps, but perhaps we should make this more explicit.

```suggestion
The more terms in a query, the longer it will take MeiliSearch to return matching documents. This goes against our goal of providing a [fast search-as-you-type experience](/learn/what_is_meilisearch/philosophy.md#front-facing-search).
```

Reply:
I'm not convinced we should go with this phrasing, since the shortest queries can sometimes take the longest (single-letter queries on large datasets, for example). I think it also probably depends whether you have the `words` ranking rule enabled. This seems like it could be misleading, but then again, I'm no expert! We could discuss it tomorrow in the docs sync meeting.

Reply:
Yeah, you're right. It's a bit of a bold statement which very likely simplifies what actually happens under the hood.

```suggestion
Queries with many search terms can lead to long response times. This goes against our goal of providing a [fast search-as-you-type experience](/learn/what_is_meilisearch/philosophy.md#front-facing-search).
```